components/gfx/font_cache_task.rs

```diff
@@ -309,11 +322,11 @@ impl FontCacheTask {
         }
     }
 
-    pub fn find_font_template(&self, family: String, desc: FontTemplateDescriptor)
+    pub fn find_font_template(&self, family: &FontFamily, desc: FontTemplateDescriptor)
                                                 -> Option<Arc<FontTemplateData>> {
 
         let (response_chan, response_port) = ipc::channel().unwrap();
-        self.chan.send(Command::GetFontTemplate(family, desc, response_chan)).unwrap();
+        self.chan.send(Command::GetFontTemplate(family.name().to_owned(), desc, response_chan)).unwrap();
```

Comment:
When I did this a while ago, I originally tried to pass around the enum FontFamily version instead of using a string, but I ran into problems with some functions that are being called from another module that depend on using a string. I will look around again to see if there is anything I can do, and I will post my findings.